python/feathub/dsl/ast.py

```diff
@@ -161,10 +197,9 @@ def to_json(self) -> Dict:
         }
 
 
-class GroupNode(ExprAST):
+class GroupNode(AbstractUnaryOp):
```

Comment:
I think we should keep the `GroupNode`. For example, in `FlinkProcessor` and `SparkProcessor`, where the Feathub Expr is translated to Flink SQL Expr and Spark SQL Expr, they can benefit from the GroupNode. Otherwise, they must put parentheses around all the binary operators in Flink SQL Expr and Spark SQL Expr.